package.json

```diff
@@ -32,7 +32,7 @@
     "chalk": "^2.3.0",
     "concat-stream": "^1.6.0",
     "find-nearest-file": "^1.1.0",
-    "meow": "^4.0.0",
+    "meow": "^10.0.1",
```

Comment:
We can't use meow v10 as it requires Node 12 — https://github.com/sindresorhus/meow/releases/tag/v10.0.0

Looks like CI doesn't actually run `bin/cli.js` so it's passing when it shouldn't be 😬

The latest we can go with Node 8 support is v6.1.1 which [requires a high enough version of trim-newlines](https://github.com/sindresorhus/meow/blob/v6.1.1/package.json#L51) to fix the warning